dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -665,7 +709,8 @@ void ScrollViewer::OnVerticalScrollControllerPointerEntered(
 
     m_isPointerOverVerticalScrollController = true;
 
-    if (!SharedHelpers::IsAnimationsEnabled())
+    UpdateScrollControllersAutoHiding();
```

Comment:
If this is a generally expected call pattern maybe have update be an argument to AreScrollControllersAutoHiding()?
(defaults to false)

Reply:
There are only two instances of calling UpdateScrollControllersAutoHiding() first out of 14 calls so I'll leave this as is.